component/sections/pricing/variant_b.js

```diff
@@ -85,12 +85,22 @@ function VariantB({ caption, title, description, plans }) {
                     {plans?.[0]?.primaryButton?.label && (
                       <a
                         className="inline-block mt-4 lg:mt-0 py-2 px-6 rounded-l-xl rounded-t-xl bg-webriq-blue hover:bg-webriq-darkblue text-white font-bold leading-loose transition duration-200"
-                        target={plans?.[0]?.primaryButton?.pageAccess === "openLinkToNewTab" ? "_blank" : null}
-                        rel={plans?.[0]?.primaryButton?.pageAccess === "openLinkToNewTab" ? "noopener noreferrer" : null}
+                        target={
+                          plans?.[0]?.primaryButton?.linkTarget ===
+                          "openLinkToNewTab"
+                            ? "_blank"
+                            : null
+                        }
+                        rel={
```

Comment:
This **_rel_** attribute value was not updated. Need to change value to
`plans?.[0]?.primaryButton?.linkTarget === "_blank" ? "noopener noreferrer" : null` .